raft/raft.go

```diff
@@ -544,10 +585,14 @@ func (r *raft) poll(id uint64, v bool) (granted int) {
 }
 
 func (r *raft) Step(m pb.Message) error {
+	sp := pbutil.StartSpanFromMessage("raft/Step", m)
```

Comment:
Ideally it would be really nice to be able to contain a live span in messages. Oh well.